src/test/java/org/cactoos/package-info.java

```diff
@@ -25,6 +25,11 @@
 /**
  * Tests.
  *
+ * @todo #1396:30min Continue replacing usage of MatcherAssert.assertThat with
+ *  Assertion from cactoos-matchers. Choose package and replace assertThat
+ *  with Assertion. Update puzzle for the next package
```

Comment:
@andreoss *Choose package and replace assertThat with Assertion.* Have you already said it on the fist phrase? To me, it seems redundant.

Reply:
@fabriciofx Done